apps/ex_nvr/lib/ex_nvr/devices/lpr_events_puller.ex

```diff
@@ -0,0 +1,55 @@
+defmodule ExNVR.Devices.LPREventsPuller do
```

Comment:
```suggestion
defmodule ExNVR.Devices.LPREventPuller do
```

Update the name of the file too.